example/basic/example.dart

```diff
@@ -21,10 +21,13 @@ main() {
         name: 'one',
         defaultRoute: true,
         path: '/one',
-        pageTitle: 'Route One',
+        pageTitle: (_) => 'Route One',
```

Comment:
I went ahead and made it dynamic, so valid values = null, String, PageTitleHandler